README.md

```diff
@@ -1,5 +1,44 @@
-# DEV1-IMU
+# DEV1 IMU
 
 ## Introduction
 
-*The Inertial Measurement Unit senses the acceleration of the bike and reports any relevant data over CAN.*
\ No newline at end of file
+### DEV1 IMU
+
+The Inertial Measurement Unit measures acceleration and position data for the
+bike using the BNO055 chip. This chip uses its sensors (
+accelerometers, magnetometer, and gyroscope) to calculate a number of
+metrics, including gravity vector, linear acceleration, angular velocity,
+Euler angles, and the surrounding magnetic field. The IMU reports these
+measurements over CAN, providing general information to the bike on its
+orientation and allowing other boards to act accordingly for optimal
```

Comment:
Took in suggestion and also:
- added the sensors and how the BNO055 uses it
- changed suggested *exposed data* to *report these measurements*  cause I think it's a little more clearer what is being done